src/openeo_gfmap/fetching/generic.py

```diff
@@ -45,23 +48,34 @@ def generic_default_fetcher(
         bands: list,
         **params,
     ) -> openeo.DataCube:
-        bands = convert_band_names(bands, BASE_MAPPING)
+        if band_mapping is not None:
+            bands = convert_band_names(bands, band_mapping)
 
-        if (collection_name == "COPERNICUS_30") and (temporal_extent is not None):
+        if (collection_name in KNOWN_UNTEMPORAL_COLLECTIONS) and (
+            temporal_extent is not None
+        ):
             _log.warning(
-                "User set-up non None temporal extent for DEM collection. Ignoring it."
+                "User set-up non None temporal extent for %s collection. Ignoring it.",
```

Comment:
"none None temporal extent" is quite confusing. DO you mean that they did set up a temporal extent?

Reply:
What do you ignore? It would be more correct to log that you are setting the temporal extent to none

Reply:
I changed this sentece, as It was completely unintelligible 😆